Movied/Movied/Scenes/Home/Cells/MovieItem.swift

```diff
@@ -0,0 +1,23 @@
+import UIKit
+
+final class MovieItem: UICollectionViewCell {
```

Comment:
Set name of cell file name and class are same, and as template as  Name+CollectionViewCell or shorter Name+Cell
So, it should be MovieItemCell or MovieItemCollectionViewCell